packages/@okta/i18n/src/properties/login_cs.properties

```diff
@@ -891,6 +1013,14 @@ oie.authenticator.oktaverify.method.totp.verify.enable.biometrics.description =
 oie.authenticator.oktaverify.method.totp.verify.enable.biometrics.point1 = Va\u0161e za\u0159\u00EDzen\u00ED podporuje biometriku.
 oie.authenticator.oktaverify.method.totp.verify.enable.biometrics.point2 = Aplikace Okta Verify je aktu\u00E1ln\u00ED.
 oie.authenticator.oktaverify.method.totp.verify.enable.biometrics.point3 = V aplikaci Okta Verify je pro v\u00E1\u0161 \u00FA\u010Det povolena biometrika.
+oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.mobile = Va\u0161e odpov\u011B\u010F byla p\u0159ijata, ale va\u0161e organizace vy\u017Eaduje biometriku. Ujist\u011Bte se, \u017Ee va\u0161e za\u0159\u00EDzen\u00ED podporuje biometriku a \u017Ee aplikace Okta Verify je aktu\u00E1ln\u00ED a m\u00E1te v n\u00ED pro sv\u016Fj \u00FA\u010Det povolenou biometriku, a pak to zkuste znovu.
+oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.desktop = Va\u0161e odpov\u011B\u010F byla p\u0159ijata, ale va\u0161e organizace vy\u017Eaduje biometriku. Ujist\u011Bte se, \u017Ee va\u0161e za\u0159\u00EDzen\u00ED podporuje biometriku a \u017Ee aplikace Okta Verify je aktu\u00E1ln\u00ED a m\u00E1te v n\u00ED pro sv\u016Fj \u00FA\u010Det povolenou biometriku a biometrick\u00E9 sn\u00EDma\u010De va\u0161eho za\u0159\u00EDzen\u00ED jsou p\u0159\u00EDstupn\u00E9. Pak to zkuste znovu.
```

Comment:
https://oktainc.atlassian.net/browse/OKTA-450852  @carloslima-okta @nikhilvenkatraman-okta @keithchin-okta 

Re-write needed for English string...,

_Make sure your device supports biometrics, Okta Verify is up-to-date, biometrics are enabled for your account in Okta Verify and your device's biometric sensors are accessible, then try again._

Issue:

Sentence length and cramming all messages into 1 sentence

Impact:

Translation quality

High risk for breaking UI

Reply:
What is the ideal solution here ? Should we break down the string and present them in a bulleted list form ?

Reply:
we are already parsing this in the SIW and display it as bullet points (see strings below) since this sentence is too long. So at least it is not as bad, but we should revisit this and see if what we have already displayed as bullet points is good i18n-wise